fl4health/clients/nnunet_client.py

```diff
@@ -201,7 +201,7 @@ def get_data_loaders(self, config: Config) -> Tuple[DataLoader, DataLoader]:
         # Set the number of processes for each dataloader.
         if self.n_dataload_proc is None:
             # Nnunet default is 12 or max cpu's. We decrease max by 1 just in case
-            self.n_dataload_proc = min(12, len(os.sched_getaffinity(0)) - 1)
+            self.n_dataload_proc = min(12, len(os.sched_getaffinity(0)) - 1)  # type: ignore
```

Comment:
I had to do this for my machine because `sched_getaffinity` is apparently not available for the older intel macs in Python 3.10 🤔. I'm not sure why. I think it's fine to leave this here, as it seems to work for you guys and I likely won't be running this until I get my new machine anyway.

Reply:
All good with me. Maybe add a short comment as to why its being ignored? Maybe worth looking into late.

Reply:
Yeah good call. Will add a comment. I looked into it a bit and there really isn't a great alternative for older MacOS setups. Most people just end up punting to getting all cpus rather than those accessible from the process thread. There is a new function in os for python 3.13 and up that seems to do this better for MacOS but it's not available below that...